src/seq_mv/HYPRE_seq_mv.h

```diff
@@ -80,6 +80,8 @@ HYPRE_Int HYPRE_VectorDestroy( HYPRE_Vector vector );
 HYPRE_Int HYPRE_VectorInitialize( HYPRE_Vector vector );
 HYPRE_Int HYPRE_VectorPrint( HYPRE_Vector vector, char *file_name );
 HYPRE_Vector HYPRE_VectorRead( char *file_name );
+HYPRE_Int HYPRE_VectorGetData( HYPRE_Vector vector, HYPRE_Complex **data);
```

Comment:
Why is this something we want to expose to users?

Reply:
This is due to how `amg.c` was written: it doesn't include any developer level head files, only user level `HYPRE_*.h` instead, which I assumed was required by the benchmark. If this is the case, we have to have some user level functions for `HYPRE_Vector` (especially for making the code portable on GPUs and simple) for which we basically don't provide anything currently, expect create and destroy (which also seems weird).

Reply:
If we are allowed to include any headers in the benchmark, I can remove all these, which basically leaves this PR nothing except for a simple mistake made in `_hypre_utilities.h`.

Reply:
> copied here from @ulrikeyang: Yes, I was mostly trying to avoid having to include _hypre .h files. I also think that the new routines can be useful for users.

Reply:
The issue I have here is that GetData() exposes the underlying data storage to the user, so we lock ourselves into a specific format if we want to be backward compatible.  In the case of a simple sequential vector, it's probably okay to assume this is just an array of doubles (HYPRE_Complex) (though there is additional complexity to consider when the vector is a true multivector).  The reason we are doing this here is to simplify the CPU/GPU complexity for setting values in a double array, which is otherwise just a trivial `for` loop (at least that's my take on what's happening in `amg.c`).

I agree that we don't want to include `_hypre_*.h` files in the benchmark driver.

Anyway, since this is an interface that isn't really widely used or advertised to users, I am willing to go along with it just to make life simple.  Thanks!

Reply:
I see your argument. We actually already have _hypre_utilities.h in the driver for the timing routines. So, we could add one more, if you want to avoid this issue.

Reply:
OK. This sounds good to me. So, I guess I can just close this PR without merging in anything. I will change `amg.c` accordingly, which will be much easier to do. Thanks again!